packages/quick_actions/quick_actions_ios/example/ios/Runner/Info.plist

```diff
@@ -45,5 +45,9 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
+	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
```

Comment:
these changes are automatically added when i do `flutter build ios`

Reply:
These should be fine I think?

Reply:
Yes, these and the `alwaysOutOfDate` change are from auto-migrations.